Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx

```diff
@@ -73,15 +66,47 @@ export default class AssignDatapoints extends React.Component {
                 >
                   {strings.edit}
                 </a>
-              </p>
-            </div>
+              </>
+            )}
+          </p>
+        </div>
 
-            <button onClick={() => this.changeTab('SEARCH_DATAPOINTS')} type="button">
-              {strings.assignDatapointByNameOrId}
-            </button>
-          </div>
+        <Dropdown disabled={data.allDataPointsAssigned} title="Assign Datapoints">
```

Comment:
Should the title also be added as a string?

Reply:
Do you mean translation?

Reply:
Yes

Reply:
Yes it should. Updating now